pages/portfolio.js

```diff
@@ -1,19 +1,96 @@
 import styled from 'styled-components';
+import Image from 'next/image';
+// why are the photos taking so much time to load?
+
 const Container = styled.div`
   position: absolute;
-  top: 0; /* Header Height */
-  bottom: 0; /* Footer Height */
+  top: 0;
+  /* bottom: 0; //Footer Height */
   width: 100%;
   display: flex;
   flex-direction: column;
   align-items: center;
   margin-top: 5em;
+  padding: 2em;
+`;
+
+// find out why photos are loading so slow!
+const X = 3;
+const Central = () => {
+  return (
+    <Image
+      src="/pics/CentralStreetA.jpg"
+      alt="A beautiful landscape by Bettina Baschy lanscapes"
+      width={2035 / X}
+      height={1570 / X}
+    />
+  );
+};
+
+const _5thAveB = () => {
+  return (
+    <Image
+      src="/pics/5thAveB.jpg"
+      alt="A beautiful finished landscape by Bettina Baschy Lanscaping"
+      width={2035 / X}
+      height={1570 / X}
+    />
+  );
```

Comment:
Could you make this into a component, and just passed in src/alt like you did elsewhere?